src/core/plugins/menu.js

```diff
@@ -217,6 +217,7 @@ class Menu {
 		let insideBlockquote = nodeIsActive(state, schema.nodes.blockquote);
 		this.paragraph = this.buildBlock(schema.nodes.paragraph, {}, insideList || insideBlockquote);
 		this.codeBlock = this.buildBlock(schema.nodes.codeBlock);
+		this.math_display = this.buildBlock(schema.nodes.math_display);
```

Comment:
Creating `math_display` doesn't properly focus the node and you have to click it again. Unclear why. Maybe we need to programmatically move selection position into the math node. I'll try to find later an example.

Reply:
Maybe [insertMathCmd](https://github.com/benrbray/prosemirror-math/blob/9cf572db4b6258a2de1c3f1d577b865116155342/src/commands/insert-math-cmd.ts) could help here.